src/components/organisms/MessageCollection/MessageCollection.jsx

```diff
@@ -0,0 +1,43 @@
+import { MessageComponent } from '../..';
+
+export const MessageCollection = () => {
+    const usersMsgs = [
```

Comment:
Create separate file for mocks, just something like Mocks.js file

Reply:
Done